pkg/suite/suite.go

```diff
@@ -155,9 +222,17 @@ func (s *suite) Run(t *testing.T, suiteName string) {
 		logger.Log("Uninstalling App %s", app.AppName)
 		err := state.GetFramework().MC().DeleteApp(state.GetContext(), *app)
 		Expect(err).NotTo(HaveOccurred())
+
+		logger.Log("Deleting workload cluster")
+		err = teardown.New(state.GetFramework()).Teardown(state.GetCluster())
```

Comment:
would it be possible to make this step somehow optional? it be useful when running locally and trying to diagnose problems

Reply:
Could you create an issue for this please? I think it's a good idea but I'd like to tackle it on its own as I think it would also benefit `cluster-test-suites`.

Reply:
sure thing https://github.com/giantswarm/giantswarm/issues/30698

if you add a check for the E2E_WC_NAME envvar for skipping the cluster creation; you could probably do the same here for now until a better solution is implemented.